test/chpldoc/compflags/version/projectVersion.no-tag.option.doc.good

```diff
@@ -0,0 +1,10 @@
+var DOCUMENTATION_OPTIONS = {
+    URL_ROOT: document.getElementById("documentation_options").getAttribute('data-url_root'),
+    VERSION: '1.0.3',
+    LANGUAGE: 'None',
+    COLLAPSE_INDEX: false,
+    FILE_SUFFIX: '.html',
+    HAS_SOURCE: true,
+    SOURCELINK_SUFFIX: '.txt',
+    NAVIGATION_WITH_KEYS: false,
+};
```

Comment:
This seems like something that wouldn't need to be updated constantly, so I would lean towards continuing to check it

Reply:
removed all other files and now the test checks only "docs/_static/documentation_options.js"